.eslintrc.js

```diff
@@ -25,5 +25,8 @@ module.exports = {
   },
   "plugins": [
     "prettier"
-  ]
+  ],
+  "ignorePatterns": [
+    ".eslintrc.js"
```

Comment:
does this stop `eslintrc.js` from linting itself? 👀

Reply:
Totally 👍 It didn't do this before and then started doing it all of the sudden. Must be VSCode's fault.